.github/workflows/check.yml

```diff
@@ -1,43 +1,87 @@
 name: Check
 
-# XXX disabled for now because we use Jenkins still, but at the time this was tested it was functional
-# on:
-#   pull_request:
-#     branches: [ master ]
+on:
+  pull_request:
```

Comment:
Tip: To test the on pull request trigger it's easiest if you push your changes to a branch in the main repository instead of your fork. Then it will already consider your changes to the workflow.